### PR TITLE
Add members view to the UI

### PR DESCRIPTION
**Pull request by jpsantosbh**
Signalwire_Call_Demo

### CODE REVIEW

**Comment by jpsantosbh** [comment]
This depends on https://github.com/signalwire/signalwire-js/pull/968

File: public/index.umd.js [hunk 1]
```diff
@@ -0,0 +1 @@
+/workspaces/signalwire-js/packages/js/dist/index.umd.js
```
Comment (iAmmar7) [inline public/index.umd.js:1]: You have mistakenly pushed the build files.

File: views/index.ejs [hunk 2]
```diff
@@ -443,7 +451,6 @@
   </script>
   <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-C6RzsynM9kWDrMNeT87bh95OGNyZPhcTNXj1NW7RuBCsyN/o0jlpcV8Qyq46cDfL" crossorigin="anonymous"></script>
   <script type="text/javascript" src="https://unpkg.com/@signalwire/js@dev"></script>
-  <script src="/full.js"></script>
```
Comment (iAmmar7) [inline views/index.ejs:446]: Same as above, probably you forgot to undo these development setups.

File: views/index.ejs [hunk 3]
```diff
@@ -249,136 +249,144 @@
...
+                  </label>
+                  <input type="range" class="form-range" oninput="rangeInputHandler(this)" min="-50" max="50" value="0"
+                    id="playbackVolume" />
+                </div>
+              </div>
             </div>
+            <!-- Call playback controls end -->
           </div>
+          <ul class="col-4 list-group" id="members">
+          </ul>
```
Comment (iAmmar7) [inline views/index.ejs:387]: Just to have consistent code comments for readability.

```suggestion

          <!-- Member List UI -->
          <ul class="col-4 list-group" id="members"></ul>
          <!-- Member List UI end -->
```

**Review by iAmmar7 — CHANGES_REQUESTED** [review]

**Review by ryanwi — APPROVED** [review]

File: public/full.js [hunk 4]
```diff
@@ -395,18 +409,29 @@ window.connect = async () => {
...
+  const startHandler = (params) => {
+    console.debug('>> call.start() resolved', params)
 
     updateUIConnected()
 
     // loadLayouts()
   }
-  joinHandler()
+
+  const startCall = async () => new Promise(async (res, rej) => {
```
[the remark this answers is not in the document]
Comment (jpsantosbh) [reply public/full.js]: I was just a rename, reverted

File: public/full.js [hunk 5]
```diff
@@ -486,6 +524,8 @@ window.connect = async () => {
       document.getElementById('playbackVolume').value = params.volume
     }
   })
+  console.log('waiting callStart')
+  await startPromise
```
[the remark this answers is not in the document]
Comment (jpsantosbh) [reply public/full.js]: reverted

**Review by iAmmar7 — APPROVED** [review]

**Pull request merged**